packages/common/package.json

```diff
@@ -12,8 +12,8 @@
     "build": "gulp build"
   },
   "dependencies": {
-    "@hawtio/core": "~4.13.0",
-    "@hawtio/kubernetes-api": "~4.11.0",
+    "@hawtio/core": "github:vinzent/hawtio-core#test",
```

Comment:
I called it WIP because of these changes that i included.

Reply:
Ah good point. I've change it to a draft PR until we released the updated dependencies.